website/content/docs/connect/cluster-peering/usage/k8s.mdx

```diff
@@ -0,0 +1,231 @@
+    spec:
+      peer:
+        secret:
+          name: "peering-token"
+          key: "data"
+          backend: "kubernetes"
+  ```
+
+  </CodeBlockConfig>
+
+   <CodeBlockConfig filename="dialer.yaml">
+
+  ```yaml
+    apiVersion: consul.hashicorp.com/v1alpha1
+    kind: PeeringDialer
+    metadata:
+      name: cluster-01 ## The name of the peer you want to connect to
+    spec:
+      peer:
+        secret:
+          name: "peering-token"
+          key: "data"
+          backend: "kubernetes"
+  ```
+
+  </CodeBlockConfig>
+  </CodeTabs>
+
+## Guidance for new users
+
+If you are new to cluster peering on Consul, the following sections contain steps to help you set up environmental variables, update the Helm chart, and install the updated Helm chart on each cluster with Consul on Kubernetes.
+
+### Assign cluster IDs to environmental variables
+
+After you provision a Kubernetes cluster and set up your kubeconfig file to manage access to multiple Kubernetes clusters, you can assign your clusters to environmental variables for future use.
+
+1. Get the context names for your Kubernetes clusters using one of these methods:
+
+    - Run the `kubectl config current-context` command to get the context for the cluster you are currently in.
+    - Run the `kubectl config get-contexts` command to get all configured contexts in your kubeconfig file.
+  
+1. Use the `kubectl` command to export the Kubernetes context names and then set them to variables. For more information on how to use kubeconfig and contexts, refer to the [Kubernetes docs on configuring access to multiple clusters](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
+
+    ```shell-session
+    $ export CLUSTER1_CONTEXT=<CONTEXT for first Kubernetes cluster>
+    $ export CLUSTER2_CONTEXT=<CONTEXT for second Kubernetes cluster>
+    ```
+
+### Update the Helm chart
+
+To use cluster peering with Consul on Kubernetes deployments, update the Helm chart with the required values. After updating the Helm chart, you can use the `consul-k8s` CLI to apply `values.yaml` to each cluster.
+
+1. In `cluster-01`, run the following commands:
+
+  ```shell-session
+    $ export HELM_RELEASE_NAME=cluster-01
+    $ helm install ${HELM_RELEASE_NAME} hashicorp/consul --create-namespace --namespace consul --version "1.0.1" --values values.yaml --set global.datacenter=dc1 --kube-context $CLUSTER1_CONTEXT
+  ```
+
+1. In `cluster-02`, run the following commands:
+
+  ```shell-session
+  $ export HELM_RELEASE_NAME=cluster-02
+  $ helm install ${HELM_RELEASE_NAME} hashicorp/consul --create-namespace --namespace consul --version "1.0.1" --values values.yaml --set global.datacenter=dc2 --kube-context $CLUSTER2_CONTEXT
+  ```
+
+## Mesh gateway configuration for Kubernetes
+
+Mesh gateways are required for cluster peering connections. To add mesh gateways to your deployment in order to use cluster peering complete the following steps:
```

Comment:
```suggestion
Mesh gateways are required for cluster peering connections. Complete the following steps to add mesh gateways to your deployment in order to use cluster peering:
```
I know you normally like the infinitive at the beginning, but balance between the clauses is a little weird.